missilewars-plugin/src/main/java/de/butzlabben/missilewars/commands/UserCommands.java

```diff
@@ -39,62 +41,19 @@
 @CommandAlias("mw|missilewars")
 public class UserCommands extends BaseCommand {
 
```

Comment:
I have mainly changed the order of the methods in the class:
1. vote
2. change
3. leave